connector/json-test-connector/src/main.rs

```diff
@@ -1,3 +1,24 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use fluvio_connector_package::config::ConnectorConfig;
+
 fn main() {
     println!("Hello, world!");
+
+    let opts = Opt::parse();
+    println!(
+        "Reading config file from: {}",
+        opts.config.to_string_lossy()
+    );
+
+    let config =
+        ConnectorConfig::from_file(opts.config.as_path()).expect("unable to read config file");
+    println!("{:#?}", config);
+}
+
```

Comment:
Can you fill in the actual logic to read from the config so can test and see generalizes it?

Reply:
What logic do you mean here? For test purposes, I read the config from a file and print it. Do you mean an assertion that the values are equal to what we expect?

Reply:
This can be done in separate PR.  Not showstopper